src/uct/ib/base/ib_md.c

```diff
@@ -11,13 +11,49 @@
 
 #include <ucs/arch/atomic.h>
 #include <pthread.h>
+#include <numaif.h>
```

Comment:
For headers you have to install the *dev.rpms. So I would say you have to check both.